assets/app/scripts/filters/util.js

```diff
@@ -105,6 +105,12 @@ angular.module('openshiftConsole')
         // We didn't get an amount? shouldn't happen but just in case
         return [value, null];
       }
+
+      // No normalization of unit.
+      if (!humanizeUnits) {
+        return split.slice(1);
```

Comment:
> So this is stripping the unit and only returning the value?

No, it's returning the amount (index 1) and unit (index 2).

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/12680436/cabbce62-c677-11e5-9f66-6a11f9881e05.png)